rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -259,7 +259,7 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
       )
     val target =
       """new x0 in {
-        |  for( @{x1} <- @{x0} ; @{x2} <- @{x0} ) {
+        |  for( @{x1} <- @{*x0} ; @{x2} <- @{*x0} ) {
```

Comment:
Please open a JIRA ticket for removing the `quote-unquote` (`@{*x}`) noise from pretty printer output

Reply:
https://rchain.atlassian.net/browse/RHOL-1054